svf/lib/AE/Svfexe/AEDetector.cpp

```diff
@@ -500,3 +506,154 @@ bool BufOverflowDetector::canSafelyAccessMemory(AbstractState& as, const SVF::SV
     }
     return true;
 }
+
+void NullptrDerefDetector::detect(AbstractState& as, const ICFGNode* node) {
+    if (!SVFUtil::isa<CallICFGNode>(node)){
```

Comment:
switch if and else

if(const CallICFGNode* callNode = SVFUtil::dyn_cast<CallICFGNode>(node))
{

}
else
{
}